files/scripts/syncd.sh

```diff
@@ -44,12 +44,19 @@ function startplatform() {
 }
 
 function waitplatform() {
-    
+
     if [[ x"$sonic_asic_platform" == x"mellanox" ]]; then
         debug "Starting pmon service..."
         /bin/systemctl start pmon
         debug "Started pmon service"
     fi
+    if [[ x"$BOOT_TYPE" != x"cold" ]]; then
```

Comment:
Minor: This is correct, but it may be better to match against whitelist BOOT_TYPEs (fast, warm, fastfast).
This is so that this condition does not evaluate to true for any other boottype that gets added in future.

Reply:
Done